service/cluster/kubernetes_cluster.go

```diff
@@ -136,6 +136,10 @@ func (k KubernetesClusterService) StartMonitoring(ctx context.Context, req *cont
 		return nil, status.Error(codes.FailedPrecondition, k.p.Sprintf("Unable to connect to Kubernetes cluster: %s", err))
 	}
 	defer k8sClient.Cleanup() //nolint:errcheck
+	err = k8sClient.CreateKubeStats(ctx)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
CreateKubeStats doesn't work without VMOperator, so I suggest moving invokation it inside CreateVMOperator.

Reply:
Okay. I got it. Thanks for the suggestion